web/styles/left_sidebar.css

```diff
@@ -626,14 +626,15 @@ li.top_left_scheduled_messages {
     All of our left sidebar handlers use absolute
     positioning.  We should fix that.
 */
-.all-messages-sidebar-menu-icon,
-.stream-sidebar-menu-icon,
-.starred-messages-sidebar-menu-icon,
-.drafts-sidebar-menu-icon,
-.topic-sidebar-menu-icon {
+.top_left_row *[class*="sidebar-menu-icon"],
+.bottom_left_row *[class*="sidebar-menu-icon"] {
```

Comment:
I would be tempted to make a `sidebar-menu-icon` CSS class instead of using attribute selectors; in part because I worry with the enormous number DOM elements we have about performance impacts of using attribute selectors, but more because I think it better expresses the idea we have here, which is that we have a "sidebar menu icon" pseudocomponent that we're defining the styles for, not 5 somewhat related components.

Reply:
Cool. Well, the hard part is done in having consolidated those selectors. Swapping out the attributes with a new `sidebar-menu-icon` should be NBD, of course. I just need to make sure I don't miss any spots in the templates.

Reply:
Okay. Updated with the `sidebar-menu-item` utility class. I caught a spot in `stream_popovers.js` that was also using a wildcard attribute selector, and I swapped that out and replaced it with the new utility class there, too, so long as I was thinking about it.